CppCoreGuidelines.md

```diff
@@ -2621,8 +2622,33 @@ For passthrough functions that pass in parameters (by ordinary reference or by p
 
 **Enforcement**: ???
 
+<a name="Rf-decltype-forward"></a>
+### F.54: Use decltype on auto&& parameters to std::forward them.
+ 
+ 
+**Reason**: One of the most exciting features of C++14 is generic lambdas—lambdas that use auto in their parameter specifications. The implementation of this feature is straight‐forward: operator() in the lambda’s closure class is a template. Given this lambda,for example :
+
+		auto f = [](auto x){ return func(normalize(x)); };
```

Comment:
indentation should be one tab or four spaces, else code will be rendered indented. Same below